src/apps/loan-list/materials/manifestation.graphql

```diff
@@ -0,0 +1,20 @@
+query getMaterialManifestation($faust: String!) {
```

Comment:
This query name is a bit to generic in the entire scope of queries. I think you need to be more specific. We are also using material manifestations in other places and maybe we do not need the same set of properties.

Reply:
All the querys are used by graphql-codegen and end in the same scope in src/core/dbc-gateway/generated/graphql.tsx.
If we are specific we will not get name collisions although we are using the same type of queries eg: `manifestation(faust: $faust) {`
Regarding properties we use fragments in order to define sets of reusable properties which the codegen tool also creates types from. But this is a subject that is more easy to explain p2p :)